### PR TITLE
Alvaro lossada group member display

### PR DESCRIPTION
**Pull request by OnlyAlf**

### CODE REVIEW

File: app/src/main/java/com/batch/mcs/finalproject/GroupMembersFragment.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link GroupMembersFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link GroupMembersFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class GroupMembersFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
```
Comment (itzcoatl90) [inline app/src/main/java/com/batch/mcs/finalproject/GroupMembersFragment.java]: Nop.... please do not inject default code...

Comment (itzcoatl90) [reply app/src/main/java/com/batch/mcs/finalproject/GroupMembersFragment.java]: Check all other fragments

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -3,12 +3,32 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_padding">8dp</dimen>
     <dimen name="activity_vertical_padding">8dp</dimen>
-    <dimen name="title_text_size">12dp</dimen>
```
Comment (itzcoatl90) [inline app/src/main/res/values/dimens.xml]: Are we sure this is not going to break the app?? please do not erase it

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -3,12 +3,32 @@
...
+    <dimen name="recycleview_chat_user_interaction_sent_cardview_tv_message_width">249dp</dimen>
+    <dimen name="recycleview_chat_user_interaction_sent_cardview_tv_timestamp_height">18dp</dimen>
+    <dimen name="recycleview_chat_user_interaction_sent_cardview_tv_timestamp_width">83dp</dimen>
+    <dimen name="recycleview_chat_user_interaction_sent_cardview_tv_timestamp_margin_right">161dp</dimen>
+    <dimen name="rv_fragment_group_members_height">463dp</dimen>
+    <dimen name="views_update_top_margin">100dp</dimen>
     <dimen name="views_update_bottom_margin">150dp</dimen>
     <dimen name="max_lines">1</dimen>
     <dimen name="no_dimension">0dp</dimen>
-    <dimen name="input_margin_start_to_end">8dp</dimen>
```
Comment (itzcoatl90) [inline app/src/main/res/values/dimens.xml]: Please do not erase it

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,5 +1,9 @@
 <resources>
     <string name="app_name">FinalProject</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (itzcoatl90) [inline app/src/main/res/values/strings.xml]: do not inject default values

**Review by itzcoatl90 — CHANGES_REQUESTED** [review]
Address comments

File: .idea/codeStyles/Project.xml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (itzcoatl90) [inline .idea/codeStyles/Project.xml:1]: Nop...

**Review by itzcoatl90 — DISMISSED** [review]
Do not add the files that are not required

**Review by ecejmsr — APPROVED** [review]
Everything looks good.

**Pull request merged**